winjs/README.md

```diff
@@ -5,7 +5,7 @@ SURFnet SSO Authentication library for Windows (WinJS)
 
 1. Copy the file [SSOManager.js](library/js/sso/SSOManager.js) inside your WinJS project (preferably inside the *Shared* project, if you are targeting multiple platforms, and intend to use SSO on all of them).
 2. Don't forget to add the file inside your Visual Studio project after copying the file in the folder. Right-click on the project, `Add -> Existing item...` and navigate to where you added the file and select it. It should appear in *the Solution Explorer* now.
-3. Now you have to register your application as an SSO client at the identity provider configuration. For this you need the callback URL of your application, which you can find out by checking the return value of `Surfnet.Nonweb.Sso.SSOManager._getInstance().getCurrentApplicationCallbackUri()`. Note that the phone and tablet/desktop application are separate applications, and have a different URL, so they need different client IDs too (so you have to register 2 clients).
+3. Now you have to register your application as an SSO client at the identity provider configuration. This will give you the ClientID's you need in step 5. For this you need the callback URL of your application, which you can find out by checking the return value of `Surfnet.Nonweb.Sso.SSOManager._getInstance().getCurrentApplicationCallbackUri()`. Note that the phone and tablet/desktop application are separate applications, and have a different URL, so they need different client IDs too (so you have to register 2 clients). The fastest way to get these is to run your application in debug, and while running, in the Javascript console, enter `Surfnet.Nonweb.Sso.SSOManager._getInstance().getCurrentApplicationCallbackUri()`. the URL should look something like `ms-app://s-1-15-2-1668544952-1132098162-3287153653-926875653-937996570-661899771-2544785182/`
```

Comment:
There's some duplication in this line with the callback uri:
"For this you need the callback URL of your application, which you can find out by checking the return value of `Surfnet.Nonweb.Sso.SSOManager._getInstance().getCurrentApplicationCallbackUri()`"

and 

"The fastest way to get these is to run your application in debug, and while running, in the Javascript console, enter `Surfnet.Nonweb.Sso.SSOManager._getInstance().getCurrentApplicationCallbackUri()`" mean mostly the same.